docs/how-to-use-gvfs.md

```diff
@@ -49,22 +49,69 @@ the path mapping and convert automatically.
+| `fs.gravitino.fileset.cache.evictionMillsAfterAccess` | The value of time that the cache expires after accessing in the Gravitino Virtual File System. The value is in `milliseconds`.                                                                          | `3600000`     | No                                                                                                    | 0.5.0            |
+| `fs.gravitino.fileset.cache.evictionMillsAfterAccess` | The value of time that the cache expires after accessing in the Gravitino Virtual File System. The value is in `milliseconds`.                                                                          | `3600000`     | No                                                                                                    | 0.5.0            |
+
+Apart from the above properties, to access fileset like S3, GCS, OSS or custom fileset through GVFS, you need to configure the following extra properties.
+
+#### S3 fileset
+
+| Configuration item             | Description                                                                                                                                                                           | Default value | Required                  | Since version    |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|---------------------------|------------------|
+| `fs.gvfs.filesystem.providers` | The file system providers to add. Set it to `s3` if it's a s3 fileset or a comma separated string that contains `s3` like `gs,s3` to support multiple kind of fileset including `s3`. | (none)        | Yes if it's a s3 fileset. | 0.7.0-incubating |
+| `s3-endpoint`                  | The endpoint of the AWS s3.                                                                                                                                                           | (none)        | Yes if it's a s3 fileset. | 0.7.0-incubating |
+| `s3-access-key-id`             | The access key of the AWS s3.                                                                                                                                                         | (none)        | Yes if it's a s3 fileset. | 0.7.0-incubating |
+| `s3-secret-access-key`         | The secret key of the AWS s3.                                                                                                                                                         | (none)        | Yes if it's a s3 fileset. | 0.7.0-incubating |
+
+At the same time, you need to place the corresponding bundle jar [gravitino-aws-bundle-{version}.jar](https://repo1.maven.org/maven2/org/apache/gravitino/aws-bundle/) in the Hadoop environment.
```

Comment:
You'd better give an example of how to use gvfs with s3 or gcs, not vaguely saying "in the Hadoop environment".

Reply:
done.